.gitlab-ci.yml

```diff
@@ -0,0 +1,203 @@
+stages:
+  - build
+  - deploy
+
+.artifacts: &artifacts
+  artifacts:
+    paths:
+      - build/
+
+.cache: &cache
+  cache:
+    key: "$CI_BUILD_NAME"
+    paths:
+      - cache/
+
+
+#================================ DEBIAN-BASED ================================
+
+.build_rc_package_deb: &build_rc_package_deb
+  stage: build
+  script:
+    - mkdir build || true # allowed to fail
+    - cd build
+    - cmake .. -DWITH_SERVER=1 -DCMAKE_BUILD_TYPE=Release -DCPACK_GENERATOR=DEB
+    - make package -j2
```

Comment:
I use -j2 here because Gitlab Runner will starve the rest of the server if I don't limit the number of jobs.